src/ui/components/pages/Info.tsx

```diff
@@ -30,62 +30,41 @@ class InfoComponent extends Component<WithTranslation> {
               <a
                 rel="noopener noreferrer"
                 target="_blank"
-                href="https://github.com/wavesplatform"
+                href="https://github.com/Keeper-Wallet"
               ></a>
             </li>
             <li className={styles.telegram}>
               <a
                 rel="noopener noreferrer"
                 target="_blank"
-                href="https://t.me/wavesnews"
-              ></a>
-            </li>
-            <li className={styles.discord}>
-              <a
-                rel="noopener noreferrer"
-                target="_blank"
-                href="https://discordapp.com/invite/cnFmDyA"
+                href="https://t.me/KeeperWallet"
```

Comment:
Let's also add the chat: https://t.me/KeeperCommunity
Probably this should have different logo.